Tests/UnitTests/Examples/Algorithms/Digitization/ModuleClustersTests.cpp

```diff
@@ -0,0 +1,85 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <boost/test/unit_test.hpp>
+
+#include "Acts/Utilities/BinningData.hpp"
+#include "ActsExamples/Digitization/ModuleClusters.hpp"
+#include "ActsFatras/Digitization/Channelizer.hpp"
+
+using namespace Acts;
+using namespace ActsFatras;
+using namespace ActsExamples;
+
+BOOST_AUTO_TEST_SUITE(DigitizationModuleClustersTests)
+
+BOOST_AUTO_TEST_CASE(merge) {
```

Comment:
One could combine these two tests to a function taking the `merge` and `expectedClusters` as arguments...

Reply:
Keep you code DRY but your tests wet?